packages/core/admin/ee/admin/pages/SettingsPage/pages/SingleSignOn/index.js

```diff
@@ -189,6 +192,48 @@ export const SingleSignOn = () => {
                       ))}
                     </Select>
                   </GridItem>
+                  <GridItem col={6} m={6} s={12}>
+                    <MultiSelect
+                      disabled={!canUpdate}
+                      hint={formatMessage({
+                        id: 'Settings.sso.form.localAuthenticationLock.description',
+                        defaultMessage:
+                          'Select the roles for which you want to disable the local authentication',
+                      })}
+                      error={
+                        formErrors.authenticationDisabled
+                          ? formatMessage({
+                              id: formErrors.authenticationDisabled.id,
+                              defaultMessage: formErrors.authenticationDisabled.id,
+                            })
+                          : ''
+                      }
+                      label={formatMessage({
+                        id: 'Settings.sso.form.localAuthenticationLock.label',
+                        defaultMessage: 'Local authentication lock-out',
+                      })}
+                      name="authenticationDisabled"
+                      onChange={(value) => {
+                        handleChange({ target: { name: 'authenticationDisabled', value } });
+                      }}
+                      placeholder={formatMessage({
+                        id: 'components.InputSelect.option.placeholder',
+                        defaultMessage: 'Choose here',
+                      })}
+                      onClear={() => {
+                        const emptyArray = [1];
+                        handleChange({ target: { name: 'authenticationDisabled', emptyArray } });;
```

Comment:
```suggestion
                        const emptyArray = [1];
                        handleChange({ target: { name: 'authenticationDisabled', emptyArray } });
```

just as a minor note: this is not an empty array ;)

Reply:
thanks man